solutions/dashboard/charts/elastic-project/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: elastic-project
+description: A Helm chart for Kubernetes
```

Comment:
Please add appropriate description according to tool

Reply:
Sure

Reply:
Done